### ISSUE TITLE
URI handler for Flatpak is not working

### ISSUE THREAD

**Issue by xoriole** [Tribler/tribler#6483]
The URI handler for magnet links is not working for the Flatpak installation of Tribler. Tribler does open but the start download dialog is not shown.

**Comment by qstokkink** [comment]
It would be nice to create a bleeding-edge flatpak build in a GitHub Action to check issues like this. I think we can make a test setup that uses "Git sources", see https://docs.flatpak.org/en/latest/module-sources.html#git-sources

Of course, that would be different from the "official" Flatpak (https://github.com/flathub/org.tribler.Tribler), which should use `github.com/Tribler/tribler/releases` URLs.

**Comment by qstokkink** [comment]
I got a Flatpak build running and I can confirm that this doesn't work. However, I also checked with the `.deb` file and this also doesn't work. Apparently our entire Debian (Ubuntu) build process is broken.

**Comment by qstokkink** [comment]
Wait, hold on. It does work with the latest `8.0.3` pre-release from tribler.org. Something may have been recently broken.

**Comment by qstokkink** [comment]
Alright, I got it working 🎉 However, I did not require any fixes to Tribler itself. I only needed changes in the `org.tribler.Tribler.yaml`.

I'll park my fix in https://github.com/qstokkink/tribler/tree/add_flatpak_testbuild if anyone wants to see how I got it working locally (note that this is now tied to my local filesystem and not the official Tribler release URL - this still has to be changed).

**CC:** @overflw when we hit the stable version of the 8 series the branch above has the basis for the required `.yaml` file, including the fix for this issue.
